docs-kits/kits/Connector Kit/Adoption View/adoption-view.md

```diff
@@ -1,6 +1,5 @@
 ---
 id: Adoption View
-title: Adoption View
```

Comment:
@bcronin90 have you checked if the title is used for the header 1? Or if it's the ID? Did you build it locally?

<img width="394" alt="image" src="https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/assets/84396022/cb1b2461-3668-4281-8c1b-21c6dc44ec88">